indico/web/client/js/legacy/libs/timetable/Management.js

```diff
@@ -586,11 +586,11 @@ type(
         style: {paddingLeft: pixels(30), paddingTop: pixels(8)},
       });
       this.fitInnerExplanationDiv.set(
-        'This changes the start and end times of the session blocks occurring on ' +
+        $T('This changes the start and end times of the session blocks occurring on ') +
```

Comment:
Concatenated strings are not properly translatable. Since we'll rewrite the timetable management code in the next release anyway, I'd rather not touch this.

Personally I'd do the same for anything else in here...